fuse_variables/include/fuse_variables/acceleration_angular_3d_stamped.h

```diff
@@ -114,8 +124,27 @@ class AccelerationAngular3DStamped : public FixedSizeVariable<3>, public Stamped
    * @param[out] stream The stream to write to. Defaults to stdout.
    */
   void print(std::ostream& stream = std::cout) const override;
+
+private:
+  // Allow Boost Serialization access to private methods
+  friend class boost::serialization::access;
+
+  /**
+   * @brief The Boost Serialize method that serializes all of the data members in to/out of the archive
+   *
+   * @param[in/out] archive - The archive object that holds the serialized class members
+   * @param[in] version - The version of the archive being read/written. Generally unused.
+   */
+  template<class Archive>
+  void serialize(Archive& archive, const unsigned int /* version */)
+  {
+    archive & boost::serialization::base_object<fuse_variables::FixedSizeVariable<3>>(*this);
```

Comment:
Maybe use `SIZE`, there's no chance to make this wrong, although I guess it'd fail at compile time either way: `archive & boost::serialization::base_object<fuse_variables::FixedSizeVariable<SIZE>>(*this);`

Reply:
Another option would be to explicitly call the base class `serialize` method:
``` c++
FixedSizeVariable<SIZE>::serialize(archive, version);
Stamped::serialize(archive, version);
```
which is shorter.

BTW I don't think you need `fuse_variables::` here.

Reply:
Due to the nature of the serialization system, calling the base class `serialize()` function is not an option. The serializer need to know about the class hierarchy, which gets registered when you use `boost::serialization::base_object`. But using `N`/`SIZE` is definitely a good call.

Reply:
Thanks for the clarification on calling the base class `serialize` method.